bump_rpm.sh

```diff
@@ -88,7 +88,10 @@ if [[ $CURRENT_VERSION != "$NEW_VERSION" ]] ; then
 		bump_spec $SPEC_FILE "${NEW_VERSION}"
 
 		TEMPLATE="$(awk '/^# template: / { print $3 }' $SPEC_FILE)"
-		if [[ $TEMPLATE == 'scl' ]] || [[ $TEMPLATE == 'nonscl' ]] || [[ -z $TEMPLATE ]]; then
+		if [[ $TEMPLATE == 'scl' ]] || [[ $TEMPLATE == 'nonscl' ]] || [[ -z $TEMPLATE ]] || grep --no-messages %scl_package $SPEC_FILE; then
+			if [[ $TEMPLATE == 'scl' ]] || [[ $TEMPLATE == 'nonscl' ]] || [[ -z $TEMPLATE ]] ; then
+				TEMPLATE=default
+			fi
```

Comment:
I don't think that's needed, the code uses `gem2rpm -t "$ROOT/gem2rpm/default.spec.erb" -o $SPEC_FILE ./*.gem` without the `$TEMPLATE` var anyway

Reply:
Oh, I intended to use `$TEMPLATE` in there sonthat's the bug. The idea was to maintain `hammer_pluging` if set

Reply:
Updated